### ISSUE TITLE
Package idea: @accounts/email

### ISSUE THREAD

**Issue by lfades** [accounts-js/accounts#494]
It works like `@accounts/password` but using only an email and the login process is made by an email verification.

**Comment by TimMikeladze** [comment]
What would the flow here be?

User sign up: 

1. User provides just their email address
2. Receives email with a confirmation url / token

User log in:

1. User provides just their email address
2. If that email address exists in the system, it is sent an email with a url / token which then logs user into their account?

**Comment by lfades** [comment]
Yes, the sign up should do the first log in too. [This guide](https://firebase.google.com/docs/auth/web/email-link-auth#complete_sign_in_with_the_email_link) from Firebase gives some good security details.

**Comment by agustif** [comment]
I might want to try a stab at this one later one, anyone working on it?

**Comment by TimMikeladze** [comment]
No one is working on it as far as I'm aware, @agustif it be great if you took a shot at it, that be great.

**Comment by agustif** [comment]
> No one is working on it as far as I'm aware, @agustif it be great if you took a shot at it, thanks.

Hey @TimMikeladze That's great, I will try it as soon as I get my basic graphql-modules password implementation working, most of our customers will be using the web very few times a year, so no need for them to remember a password.

**Comment by agustif** [comment]
Hey, before starting this, what about an @accounts/passwordless which can both handle (E-mail, OTP, or something like SMS, etc), which could be modular.

Does this make sense? Or should I go for separate things and do first an @accounts/email for a passwordless login setup, and then maybe add another layer in order to be able to either support a different or multiple transport to send/recieve the one time password.

**Comment by TimMikeladze** [comment]
@agustif I like the sound of `@accounts/passwordless` which can support multiple transports (email, sms, etc). Makes sense.

**Comment by durdenx** [comment]
Any news on that?

**Comment by agustif** [comment]
@durdenx A big refactor for MFA is coming, that should enable this. I think @davidyaha is working in some magic link implementation for one of his projects. So this will come in the future for sure!

**Comment by nephix** [comment]
Any updates on this?
